libpod/container_api.go

```diff
@@ -807,11 +807,28 @@ type ContainerCheckpointOptions struct {
 	// TCPEstablished tells the API to checkpoint a container
 	// even if it contains established TCP connections
 	TCPEstablished bool
+	// Export tells the API to write the checkpoint image to
+	// the filename set in Export
+	Export string
```

Comment:
Can we combine these two into one? Call it `TargetFile` and say that it's the destination for Checkpoint and the source for Restore?

Reply:
Combine export/import specifically

Reply:
done